### ISSUE TITLE
Timestamps in spreadsheets

### ISSUE THREAD

**Issue by joe-pitt** [mrghg/agage-archive#87]
The spreadsheet timestamps are interpreted as the start point of the sampling (as in the final files), but the timestamps in GCWerks and GCCompare and the mid-point. This can be 20-40 mins different. Not a huge problem in most cases, but if using data_exclude.xlsx to remove bad data points (as I am currently for the CMN ADS) this can lead to the wrong points being removed, if you specify the flags based on the GCCompare times. Would it be better to interpret the spreadsheet times as midpoints and adjust them similar to the actual data?

**Comment by joe-pitt** [comment]
See also issue #91 . So it isn't as simple as the start vs midpoint. But I still think it would be better if the spreadsheet times were aligned with the timestamps in gcwerks rather than the corrected timestamps in the output files

**Comment by mrghg** [comment]
Can we remove this from the 0.1 release, and come back to this another time @joe-pitt? Won't it require all the timestamps in the files to be adjusted?

**Comment by joe-pitt** [comment]
Yes let's come back to it. I think if we want other people to edit the files then we should make this change, but for now it is only us editing the files so it is not urgent.
